src/components/atoms/typography/ms-typography.ts

```diff
@@ -0,0 +1,57 @@
+import { LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+import { WithTheme } from '~/mixins'
+
+import { msCheckboxStyles } from './ms-typography.styles'
+
+@customElement('ms-typography')
+export class MSTypography extends WithTheme(LitElement) {
+  static styles = msCheckboxStyles
+
+  @property({ type: String })
+  id = ''
+
+  //@TODO: set to specific strings
+  // 'center' | 'inherit' | 'justify' | 'left' | 'right'
+
+  @property({ type: String })
+  align = 'inherit'
+
+  @property({ type: String })
+  color = ''
+
+  @property({ type: String })
+  weight = ''
+
+  @property({ type: String })
+  variant = ''
+
+  render() {
+    this.style.textAlign = this.align
+    this.style.fontWeight = this.weight
+    this.style.color = this.color
+
+    return this.createTypography()
+  }
+
+  createTypography() {
+    if (!this.variant && this.variant === '') return
+
+    const elementAlreadyCreatted = this.shadowRoot?.getElementById(this.id)
```

Comment:
Another approach to ensure element doesn't have children could be something like:
```
const isEmptyElement = this.shadowRoot.childen.length === 0
if(!isEmptyElement) this.shadowRoot.innerHTML = ''
const typographyElement =  Object.assign(document.createElement(`${this.variant}`), {
      innerHTML: '<slot></slot>',
    })
 .... rest of code
```
with this way you don't have to use id to get element